index.html

```diff
@@ -348,6 +350,32 @@ <h2 class="section-title">Sponsors</h2>
 	  </div>
 	</section>
 
+<!-- ## CODE OF CONDUCT SECTION ## -->
+	<section class="code-of-conduct-section" id="code-of-conduct">
+		<div class="qcContainer qcTabTitle clearfix">
+			<div class="row middle-sm">
+				<div class="col-xs-12 col-sm-6">
+					<h2 class="section-title">Code of Conduct</h2>
+				</div>
+				<div class="col-xs-12 col-sm-6 section-button-cta">
+					<a href="code-of-conduct.html" target="_blank" class="button secondary">Read the full Code of Conduct</a>
+				</div>
+			</div>
+		</div>
+		<div class="hentry qcContainer">
+			<div class="row">
+				<div class="col-xs-12 col-sm-7">
+					<h3>The Quick Version</h3>
+					<p>Our event and its associated online spaces are dedicated to providing a harassment-free experience for everyone, regardless of gender and gender identity, age, sexual orientation, disability, physical appearance, body size, race, or religion (or lack thereof). We do not tolerate harassment of participants in any form. Sexual language and imagery is not appropriate at any point during the event. Participants violating these rules may be sanctioned or expelled at the discretion of the organizers.</p>
```

Comment:
Can we get the line height here & in the Need Help? section to match what we have in, say, the sponsor blurb?